docs/getting-started/installation.md

```diff
@@ -1,54 +1,85 @@
 ---
 title: Installation
-description: ⚡ An advanced view on how to install Spicetify.
+description: ⚡ A simple guide to installing Spicetify on any platform.
 ---
 
+:::tip
+Please read our [**post-installation**](./usage.md#post-installation) guide after installing for information on Marketplace and basic setup.
+:::
+
 ## Windows
 
-### Powershell (pre-built binary) - Recommended
+### Powershell - Recommended
 
 ```powershell
 iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install.ps1 | iex
 ```
 
 ### Chocolatey
 
-Follow this guide: https://chocolatey.org/packages/spicetify-cli
+```
+choco install spicetify-cli
+```
+
+### Winget
+
+```powershell
+winget install Spicetify.Spicetify
+```
 
 ### Scoop
 
+
 ```powershell
 scoop install spicetify-cli
 ```
 
-#### Spotify installed from Scoop
+:::note
 
-- To find the location of your Spotify installation, run `scoop prefix spotify`.
+If your Spotify has also been installed through Scoop, find the location of your Spotify installation by running:
 
 ```console
 $ scoop prefix spotify
 C:\Users\<username>\scoop\apps\spotify\current
 ```
 
-After you have located it, set `spotify_path` to that directory in Spicetify's config file:
+After you have located it, set your `spotify_path` to that directory in Spicetify's [**config**](./customization/config-file) file:
 
-![scoop-spotify-path](https://user-images.githubusercontent.com/56180050/158084602-99428adf-93bb-4983-968f-14e1f4f5b253.png)
+<details>
+<summary>Example Image</summary>
+<img src="https://user-images.githubusercontent.com/56180050/158084602-99428adf-93bb-4983-968f-14e1f4f5b253.png"></img>
+</details>
+:::
 
-### Winget
 
-```powershell
-winget install Spicetify.Spicetify
+## Linux and macOS
+
+:::caution
+
+You **cannot** use Spotify installed via Snap, please re-install it using the [**debian package**](https://www.spotify.com/us/download/linux/).
+
+Flatpak Spotify users need to do some [**additional setup**](#flatpak-spotify).
+:::
+
+:::note
+
+Before applying Spicetify, you need to gain write permission on Spotify files, by running the commands:
```

Comment:
Although I get it, I honestly would move this down. 
<img width="880" alt="image" src="https://github.com/spicetify/spicetify-docs/assets/19473034/36585ebe-1e72-446c-bbaa-0c59dfd76913">
I'm immediately bombarded with a warning as soon as I reach the Linux and macOS section and it feels a bit too much.

Reply:
whilst I understand your sentiment I believe this is important since when it was at the bottom before no one would read it, we constantly have people on the support threads using snap!

Reply:
@ohitstom I know what you mean, but having it before the recommended installation instruction presents a lot of attrition. Installation should be presented in the most simple way possible. Let's compromise though. 

This section applies to both OSs, right?
<img width="713" alt="image" src="https://github.com/spicetify/spicetify-docs/assets/19473034/90f654b3-4a23-4d2f-b85b-e230980dab28">
Let's keep this one, but move the snap warning lower. Linux users are generally more advanced users and should be able to scroll down a bit 😅

Reply:
I really dont think its a good idea to move the warning lower since if they miss it they have to completely uninstall both spotify and spicetify with the snap stuff in mind. However i have moved the note to the post installation guide so i hope this is enough, i'd be happy to change it from a warning to something else but since this is something users will need to know BEFORE installing spicetify i think its important that its above.